code/processes/orchestrator.q

```diff
@@ -0,0 +1,56 @@
+/TorQ Orchestrator Process
+
+\d .orch
+
+/default parameters
+
+scalingdetails:([] time:`timestamp$(); procname:`$(); dir:`$(); instancecreated:`$(); instanceremoved:`$(); totalnumofinstances:`int$(); lowerlimit:`int$(); upperlimit:`int$());	/table for tracking scaling
+
+processlimitscsv:hsym first .proc.getconfigfile"processlimits.csv";	/location of csv file
+limits:1!("SII";enlist ",")0:processlimitscsv;	/table of scalable processes and the max number of instances allowed for each
+scaleprocslist:exec procname from limits;	/list of scalable processes
+
+/initialises connection to discovery process and creates keyed table containing the number of instances of each scalable process
+getscaleprocsinstances:{[] 
+	.servers.startup[];
+	`.orch.procs set string@/:exec procname from .servers.procstab;
```

Comment:
Why are these stringed?

Reply:
In the next line (Line 17) there is a lambda which gets the count of instances for each scalable process. This is done by checking which of the strings in the .orch.procs list are like the scalable process. E.g. {sum procs like "rdb1*"} will recognize all instances of rdb1 i.e. rdb1.2 rdb1.3 and so on

Reply:
what if there's lots of rdbs and we have an `rdb10`, would that be detected as a replicated instance of `rdb1`?

I also don't think you need the `@/:` here?

Reply:
Yes currently it would be detected so will need fixed and yes it's not needed, I see now 'like' accepts string or symbol in right operand. Thanks